apps/docs/src/development/theming.md

```diff
@@ -68,3 +68,17 @@ const { store: colorScheme } = useColorMode();
 ```
 
 Alternatively, you can use the [OnyxColorSchemeDialog](https://storybook.onyx.schwarz/?path=/docs/support-colorschemedialog--docs) component to build your own custom component.
+
+### Transistion
```

Comment:
I would document this per default, so maybe we can include this into the docs above that documents `useColorMode()`

Reply:
changed it